client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -114,8 +120,19 @@ var CreditCardPaymentBox = React.createClass( {
 					cart={ this.props.cart }
 					transactionStep={ this.props.transactionStep } />
 
-				{ cartValues.isPayPalExpressEnabled( cart )
-					? <a className={ paypalButtonClasses } href="" onClick={ this.handleToggle }>
+				{ this.props.paymentMethods > 1
+					? <span>{ this.props.translate( 'Pay with:' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Pay with' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).